evap/results/exporters.py

```diff
@@ -77,6 +77,34 @@ def deviation_to_style(self, deviation, total=False):
             style_name += "_total"
         return style_name
 
+    def filter_text_and_heading_questions(self, questions):
+        filtered_questions = []
+        current_heading_question = None
+        questions_in_current_heading = []
+
+        for question in questions:
```

Comment:
i would suggest adding a comment here that briefly explains what the following lines do